extensions/azurePublish-new/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
For the extension folder name, we should go with full camelCase - azurePublishNew